landlab/components/landslides/tests/test_landslide.py

```diff
@@ -1,5 +1,5 @@
 """
-Unit tests for landlab.components.landslides.landslide
+Unit tests for landlab.components.landslides.landslide_probability
```

Comment:
Actually Done! - left out data driven option because, it can get very messy. Other three input methods now have unit tests!